pkgs/development/python-modules/amdgpu-fan/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, numpy, pyyaml, setuptools }:
+
+buildPythonPackage rec {
+  pname = "amdgpu-fan";
+  version = "0.0.6";
+
```

Comment:
this still needs an entry in `python-modules`, which will create a python package.

The `all-packages` entry just chooses an interpreter version and makes it into a stand alone application